test/models/pregnancy_test.rb

```diff
@@ -57,35 +57,35 @@ class PregnancyTest < ActiveSupport::TestCase
 
     describe 'status method' do
       it 'should default to "No Contact Made" when a pregnancy has no calls' do
-        assert_equal 'No Contact Made', @pregnancy.status
+        assert_equal Pregnancy::STATUSES[:no_contact], @pregnancy.status
```

Comment:
Bumping all the tests to reflect your refactor is positively heroic. Thanks a ton for taking care of that!